src/utils/pino.ts

```diff
@@ -54,7 +54,7 @@ const prettyPrint = {
       let alertMsg = ''
       let notifMsg = ''
 
-      const probeMsg = `${log.iteration} id:${log.id} ${log.responseCode} ${log.url} ${log.responseTime}ms`
+      const probeMsg = `${log.iteration} id:${log.id} ${log.responseCode} ${log.method} ${log.url} ${log.responseTime}ms`
```

Comment:
cosmetic comment, feel free to ignore/resolve immediately: its kind of useless having `method` without `url`/endpoint in this log/message.

Reply:
The URL is there? In ${log.url}?

Reply:
ups, my bad.